.dockerignore

```diff
@@ -0,0 +1,3 @@
+build
+.vs
+.vscode
```

Comment:
It would be good to add the `.git` directory here. It contains many files that do not influence the Docker build context or the build result. 

On my machine the size of the Docker build context went from 234MB to 48 MB with this change. 

Maybe more files/directories can be excluded. See `.gitignore`.